src/session.ts

```diff
@@ -464,12 +468,58 @@ async function setLocalHold(session: WebPhoneSession, flag: boolean): Promise<an
 }
 
 /*--------------------------------------------------------------------------------------------------------------------*/
+function getIncomingInfoContent(this: WebPhoneSession, request): any {
+    if (!request || !request.body) {
+        return {};
+    }
+    let ret = {};
+    try {
+        ret = JSON.parse(request.body);
+    } catch (e) {
+        return {}
+    }
+    return ret;
+}
+
+function sendMoveResponse(this: WebPhoneSession,
+                          reqId: number,
+                          code: number,
+                          description: string,
+                          options: any = {}) {
+    const extraHeaders = (options.extraHeaders || [])
+            .concat(this.ua.defaultHeaders)
+            .concat(['Content-Type: application/json;charset=utf-8']);
+    this.sendRequest(C.INFO, {
+        body: JSON.stringify(
+            {response: {
+                reqId: reqId,
+                command: 'move',
+                result: {
+                    code: code,
+                    description: description
+                }
+            }
+        }),
+        extraHeaders});
+}
 
 function receiveRequest(this: WebPhoneSession, request): any {
     switch (request.method) {
         case C.INFO:
+            // For the Move2RCV request from server
+            const content = this.getIncomingInfoContent(request);
+            if (content
+            && content.request
+            && content.request.reqId
+            && content.request.command === 'move'
+            && content.request.target === 'rcv') {
+                request.reply(200);
+                this.emit('RC_MOVE_TO_RCV', content.request);
```

Comment:
We use `camelCase` for event names.

Btw, @vyshakhbabji it would be great to gather all events as `enum` and export them.